src/Components/Visualizer/Visualizer.js

```diff
@@ -254,12 +261,13 @@ class Visualizer extends React.Component {
     if (_.every(values, Number)) {
       this.setState({ currentPhase: values });
       return;
+    } else {
+      alertMessage(MESSAGES.ONLY_NUMBERS);
```

Comment:
there is a visual defect where if a input has multiple incorrect criteria, multiple alert messages pop up. the second and so on appear for a few milliseconds before disappearing.

example: enter `[1,2,3],5,4]`

Maybe we should bundle the messages together? Or fire sequential alerts in a delay

Reply:
hmm what do you mean by bundling them together, like two messages at the same time or a new message for that use case? i don't love the delay idea but i guess if it was very short it would basically look like two at the same time

Reply:
something like taking the text and accumulating it, and then sending the alert with all of the text combined.

Reply:
I actually think just the second message makes sense here. modified the regex check to include a bunch of random symbols so for some inputs it'll hit that and send the only numbers alert, but in other cases where maybe the list is input in () or like in your above use case it'll prompt the default message